packages/lodestar/src/chain/eventHandlers.ts

```diff
@@ -259,7 +261,17 @@ export async function onBlock(
         return this.attestationProcessor.processAttestationJob(job);
       }),
     ]);
+
+    if (this.metrics) {
+      // Register attestations in metrics
+      for (const attestation of indexedAttestations) {
+        if (attestation) {
+          this.metrics.registerAttestationInBlock(attestation, block.message);
```

Comment:
I think we will likely want to register attestations while processing attestations in block processing (since this will save us one iteration thru tree-backed attestations).

This should be fine for now tho. We can benchmark and move from there.